recipe/meta.yaml

```
@@ -15,7 +15,7 @@ source:
     - pyproject.patch
 
 build:
-  number: 0
+  number: 1 
```

Minor suggestion to drop the trailing blank space

```suggestion
  number: 1
```